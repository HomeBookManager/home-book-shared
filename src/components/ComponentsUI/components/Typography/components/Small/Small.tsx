import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TypographyFontType } from '../../enums';
import { TOmittedTypographyProps } from '../../types';

const Small: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.small} {...restProps}>
    {children}
  </Typography>
);

export default Small;
