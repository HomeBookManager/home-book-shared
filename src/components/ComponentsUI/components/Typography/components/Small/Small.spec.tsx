import { render } from '@testing-library/react';

// components
import Small from './Small';

describe('Small snapshots', () => {
  it('should render Small', () => {
    //before
    const { asFragment } = render(<Small>Small</Small>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
