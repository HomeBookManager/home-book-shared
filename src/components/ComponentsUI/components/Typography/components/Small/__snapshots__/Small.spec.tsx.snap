// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Small snapshots should render Small 1`] = `
<DocumentFragment>
  <small
    class="Typography Typography--regular Typography--normal Typography--small"
    data-test-text=""
  >
    Small
  </small>
</DocumentFragment>
`;
